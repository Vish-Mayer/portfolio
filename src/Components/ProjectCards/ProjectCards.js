import React, {Component} from 'react'
import { catchClause } from '@babel/types'
import ProjectCardUI from '../ProjectCardUI/ProjectCardUI'

class ProjectCards extends Component {
 
  render(){
    return(
      <div className="container-fluid d-flex justify-content-center">
        <div className="row">
          <div className="col-md-4">
            <ProjectCardUI />
          </div>
          <div className="col-md-4">
            <ProjectCardUI />
          </div>
          <div className="col-md-4">
            <ProjectCardUI />
          </div>
        </div>
      </div>
    );
  }
}

export default ProjectCards