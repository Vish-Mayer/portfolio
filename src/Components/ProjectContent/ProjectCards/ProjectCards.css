.project-cards{
  display: flex;
  justify-content: center;
}

.col-md-5{
  padding: 0.5em;
}

@media only screen and (max-width: 650px) {
  .col-md-5{
   padding: 0em;
  }
}
