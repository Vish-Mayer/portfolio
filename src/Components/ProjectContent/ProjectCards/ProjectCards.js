import React, {Component} from 'react'
import ProjectCardUI from './ProjectCardUI/ProjectCardUI'
import img1 from '../../../Assets/testimage.png'
import './ProjectCards.css'

class ProjectCards extends Component {
 
  render(){
    return(
      <div className="container-fluid d-flex justify-content-center project-cards">
        <div className="row">
          <div className="col-md-4">
            <ProjectCardUI 
              imgsrc={img1} 
              cardName="This is a Dynamic Card" 
              cardDescription= "This is a dynamic card description"/>
          </div>
          <div className="col-md-4">
          < ProjectCardUI 
              imgsrc={img1} 
              cardName="This is a Dynamic Card" 
              cardDescription= "This is a dynamic card description"/>
          </div>
          <div className="col-md-4">
            <ProjectCardUI 
              imgsrc={img1} 
              cardName="This is a Dynamic Card" 
              cardDescription= "This is a dynamic card description"/>
          </div>
        </div>
      </div>
    );
  }
}

export default ProjectCards