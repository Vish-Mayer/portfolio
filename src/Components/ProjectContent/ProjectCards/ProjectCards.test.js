import React from 'react'
import ProjectCards from './ProjectCards'

import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

describe('Projects', () => {

  let wrapper;
  
  beforeEach(() => {
    wrapper = shallow(<ProjectCards  />)
  });

  it('project cards class matches snapshot', () => {
    const output = wrapper.find('.project-cards').props('className');
    expect(JSON.stringify(output)).toMatchSnapshot();
  });
});