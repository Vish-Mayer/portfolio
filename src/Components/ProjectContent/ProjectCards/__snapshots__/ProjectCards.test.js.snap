// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Projects project cards class matches snapshot 1`] = `"{\\"className\\":\\"justify-content-center d-flex project-cards\\",\\"children\\":{\\"type\\":\\"div\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"row\\",\\"children\\":[{\\"type\\":\\"div\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"col-md-1\\"},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"div\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"col-md-5 single-card\\",\\"children\\":{\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"id\\":\\"project1\\",\\"imgsrc\\":\\"testimage.png\\",\\"imgsrc2\\":\\"background1.jpg\\",\\"imgsrc3\\":\\"testimage.png\\",\\"imgsrc4\\":\\"background1.jpg\\",\\"cardName\\":\\"This is a Dynamic Card\\",\\"cardDescription\\":\\"This is a dynamic card description This is a dynamic card description This is a dynamic card description This is a dynamic card description This is a dynamic card description This is a dynamic card description This is a dynamic card description This is a dynamic card description This is a dynamic card description This is a dynamic card description This is a dynamic card description This is a dynamic card description This is a dynamic card description This is a dynamic card description\\"},\\"_owner\\":null,\\"_store\\":{}}},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"div\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"col-md-5\\",\\"children\\":{\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"id\\":\\"project2\\",\\"imgsrc\\":\\"testimage.png\\",\\"imgsrc2\\":\\"background1.jpg\\",\\"imgsrc3\\":\\"testimage.png\\",\\"imgsrc4\\":\\"background1.jpg\\",\\"cardName\\":\\"This is a Dynamic Card\\",\\"cardDescription\\":\\"This is a dynamic card description\\"},\\"_owner\\":null,\\"_store\\":{}}},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"div\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"col-md-1\\"},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"div\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"col-md-1\\"},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"div\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"col-md-5\\",\\"children\\":{\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"id\\":\\"project3\\",\\"imgsrc\\":\\"testimage.png\\",\\"imgsrc2\\":\\"background1.jpg\\",\\"imgsrc3\\":\\"testimage.png\\",\\"imgsrc4\\":\\"background1.jpg\\",\\"cardName\\":\\"This is a Dynamic Card\\",\\"cardDescription\\":\\"This is a dynamic card description\\"},\\"_owner\\":null,\\"_store\\":{}}},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"div\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"col-md-5\\",\\"children\\":{\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"id\\":\\"project4\\",\\"imgsrc\\":\\"testimage.png\\",\\"imgsrc2\\":\\"background1.jpg\\",\\"imgsrc3\\":\\"testimage.png\\",\\"imgsrc4\\":\\"background1.jpg\\",\\"cardName\\":\\"This is a Dynamic Card\\",\\"cardDescription\\":\\"This is a dynamic card description\\"},\\"_owner\\":null,\\"_store\\":{}}},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"div\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"col-md-1\\"},\\"_owner\\":null,\\"_store\\":{}}]},\\"_owner\\":null,\\"_store\\":{}}}"`;
